### ISSUE TITLE
Document valid values for `-Sound` argument

### ISSUE THREAD

**Issue by SkyLined** [Windos/BurntToast#84]
It is not obvious what values are acceptable for the `-Sound` argument; I expect there will be some list of sound identifiers defined by the OS as it does not appear to be a path. It would be nice if the documentation listed those identifiers or linked to a webpage that provides this list.

**Comment by SkyLined** [comment]
I just noticed that if you provide an invalid value, the error message will list what appears to be valid values:
```
Default,IM,Mail,Reminder,SMS,Alarm,Alarm2,Alarm3,Alarm4,Alarm5,Alarm6,Alarm7,Alarm8,Alarm9,Alarm10,Call,Call2,Call3,Call4,Call5,Call6,Call7,Call8,Call9,Call10
```

**Comment by Windos** [comment]
Tab completion and/or CTRL + Space should also allow you to select the one you want.

![ToastTab](https://user-images.githubusercontent.com/6955786/70615397-34be7280-1c71-11ea-8ff8-1fd488abbb8b.gif)

Your absolutely right though that this should be added to the documentation.

It was once [possible to do custom audio](https://toastit.dev/2018/04/02/crouton-sounds/) by pointing to a sound file on disk, but new versions of Windows 10 introduced an issue which broke this and it was *very* low on the priority list to fix (see #74)
